src/arch/x86/x86_64/Cargo.toml

```diff
@@ -3,8 +3,10 @@ name = "arch"
 version = "0.1.0"
 authors = ["Oreboot Authors"]
 edition = "2018"
+consts = { path = "../../../lib/consts"}
```

Comment:
That shouldn't be here... or should it? I wonder why CI wouldn't complain. Maybe anything can be put here without restrictive validation.